libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -616,6 +640,32 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
         .gd-list-item {
             font-family: $gd-font-primary;
             font-weight: 400;
+
+            .gd-icon {
```

Comment:
You are duplicating some css classes. Can we move them out side the `gd-header-project-wrapper` or `gd-header-menu-item` so that they can re-useable?
```css
.gd-list-item {
    .gd-icon {
      ...
    }
    .gd-icon-slack {
        ...
     }
     ....
}
.gd-header-menu-item {
    ....
}
.gd-header-project-wrapper {
    ....
}
```